jmeter-courses.csv

```diff
@@ -0,0 +1,3 @@
+courseCode,providerCode
+3285,1JA
+V252,1JA
```

Comment:
These just happen to be courses in my local environment that had the following characteristics, respectively:
- Only one course option, so gets added immediately after confirming 
- Several course options, such that candidate sees study mode and site selection screens

We should decide which courses should go in here and how many we need in total prior to running on the load test environment.

Reply:
So i think those two journeys definitely make sense to test as they roughly average to all the journeys that would take place. 

We can definitely add many of each type, but i'm not overly convinced there's a need, as all we're really after is simulating load, which would be the same regardless of how many of each type of course we add.

Wdyt?

Reply:
>We can definitely add many of each type, but i'm not overly convinced there's a need

Yep, inclined to agree. All that's needed then is confirming the codes of two courses that match the characteristics above. Will look on the load test environment now and update this CSV if needed.